src/App.php

```diff
@@ -133,15 +133,13 @@ public function run($request = null, $response = null)
     /**
      * Handle Route.
      *
-     * @param RouteInfoInterface $routeInfo
+     * @param mixed $routeInfo
      * @param EventInterface $event
      *
      * @throws RuntimeException if dispatch does not return RouteInfo object.
      */
-    private function handleRoute(
-        RouteInfoInterface $routeInfo,
-        EventInterface $event
-    ) {
+    private function handleRoute($routeInfo, EventInterface $event)
+    {
         if (!$routeInfo instanceof RouteInfoInterface) {
```

Comment:
this check won't work if the Type hint is RouteInfoInterface, that's why `@param mixed` is passed. /cc @gianarb @EmanueleMinotto